site/en/user_guide.md

```diff
@@ -198,23 +198,23 @@ ways make use of the following syntax:
 ```sh
 -I Input-Group    # folder of the files to be processed
 -O Output-Group   # folder for the output of your processor
--p parameter.json # indication of parameters for a particular processor
+-P parameter      # indication of parameters for a particular processor
```

Comment:
`-P KEY VALUE` ?

Reply:
At first I wanted to write key value, but everywhere else it already said param value. rather change or leave it?

Reply:
Maybe add an introductory note:

> **NOTE:** The `-P` option accepts a parameter name and a parameter value. When we write `-P parameter`, we mean that `parameter` consists of `parameter name` and `parameter value`.
?

Reply:
sounds good to me :+1: